dashboard/test/controllers/maker_controller_test.rb

```diff
@@ -66,36 +85,6 @@ class MakerControllerTest < ActionController::TestCase
     assert_equal @csd6_2019, MakerController.maker_script(@student)
   end
 
-  test "shows pilot script if student is enrolled" do
-    @pilot_script = create(:script, name: 'csd-pilot-test', family_name: 'csd6', version_year: '2022', is_maker_unit: true, pilot_experiment: 'my-experiment').tap do |script|
-      lesson_group = create :lesson_group, script: script
-      lesson = create :lesson, script: script, lesson_group: lesson_group
-      create :script_level, script: script, lesson: lesson
-    end
-
-    @pilot_student = create :student
-    experiment = create :single_user_experiment, min_user_id: @pilot_student.id, name: 'my-experiment'
-    create :user_script, user: @pilot_student, script: @pilot_script, assigned_at: Time.now
-    assert_includes @pilot_student.scripts, @pilot_script
-
-    assert_equal @pilot_script, MakerController.maker_script(@pilot_student)
-    experiment.destroy
-  end
-
-  test "does not show pilot script if student is not in experiment" do
-    @pilot_script = create(:script, name: 'csd-pilot-test', family_name: 'csd6', version_year: '2022', is_maker_unit: true, pilot_experiment: 'my-experiment').tap do |script|
-      lesson_group = create :lesson_group, script: script
-      lesson = create :lesson, script: script, lesson_group: lesson_group
-      create :script_level, script: script, lesson: lesson
-    end
-
-    ## Even creating the user_script is not sufficient: the student must be enrolled in the pilot for it to show up
-    @pilot_student = create :student
-    create :user_script, user: @pilot_student, script: @pilot_script, assigned_at: Time.now
-
-    assert_equal @csd6_2019, MakerController.maker_script(@pilot_student)
-  end
```

Comment:
why are these being removed?

Reply:
> I also removed checking pilots as part of the Maker Units because we don't pilot using or annual course offerings so a pilot would not be in the 'csd' CourseOffering.

Reply:
👋 Hi! Eavesdropping again! Again not my lane, but I worry that this might undo a change that LP did for me this past year when we were piloting changes to Unit 6 and we needed to make sure the pilot appeared for pilot teachers and their students in the maker app. [Here's the Slack thread](https://codedotorg.slack.com/archives/C0T0EGE8L/p1645735437170829) with the request and change (and a teacher testimonial after the change went live).

Maybe this is all moot as we work towards depreciating the Maker App, but since we're not there yet and still assuming this is the app that teachers and students would use: I just want to flag that having pilots work correctly in the Maker App (as LP helped with) is important and hopefully not something we're losing here.

Reply:
Hey Dan - I should not have any impact on being able to pilot a unit that uses maker. The `is_maker_unit` flag right now is only used to determine which unit we show someone on the maker homepage.